### ISSUE TITLE
About implement details

### ISSUE THREAD

**Issue by woldier** [VLTSeg/VLTSeg#4]
Nice jobs! @Joris-Kuehl-TU-Berlin  

In the original paper, the author introduced a PLM loss, the total loss is PTM_loss + M2F_loss.
But in this implementation repo, I just find the M2F_loss are used.

**Comment by Joris-Kuehl-TU-Berlin** [comment]
Hi @woldier, unfortunately I'm not quite sure what you mean. You are right that the loss used in this implementation is the Mask2Former loss, unmodified. I believe that is in accord with the VLTSeg paper in its latest version. Could you point me to the particular section of the paper that you are referring to?

**Comment by woldier** [comment]
I noticed that VLTSge uses another loss function in addition to the M2F loss. As mentioned in the original paper:

`3.3. Vision-Language Guidance for Domain Generalization (VLTSeg)`
`...`
`Putting all together, we obtain the following final loss function for the downstream training of our VLTSeg approach:`
<img width="1126" height="126" alt="Image" src="https://github.com/user-attachments/assets/2d88372b-3c01-4eab-8272-d4d12226c6df" />

**Comment by Joris-Kuehl-TU-Berlin** [comment]
Ah, I see the issue. That is from an older pre-print version of the paper, when the VLM's text encoder was still used during training. You are likely looking at either [v1] or [v2] on arXiv. The PTM loss (and the text encoder with it) has been removed since pre-print [v3], as well as from the version published at ACCV. It was found that using just the vision encoder + M2F head was simpler _and_ performed better. This repo implements the latest version of the paper.

**Comment by woldier** [comment]
Thank you for your patient response. 
Have a great day.
